add-on/TA-Demisto/bin/ta_demisto/modalert_create_xsoar_incident_helper.py

```diff
@@ -72,8 +72,7 @@ def process_event(helper, *args, **kwargs):
                     method='POST',
                     headers=headers,
                     payload=incident,
-                    verify=verify,
-                    cert=ssl_cert_tmp,
+                    verify=ssl_cert_tmp if verify else False,
```

Comment:
Can't SSL cert temp be none even when verify is true?

Reply:
It can be None, and it's the user's responsibility to provide the certificate if the "Validate SSL" checkbox is marked. If it's not provided, an SSLError is returned from the request.